packages/gatsby-plugin-image/src/components/lazy-hydrate.tsx

```diff
@@ -57,20 +76,22 @@ export function lazyHydrate(
             layout,
             width,
             height,
-            backgroundColor
+            wrapperBackgroundColor
           )}
         />
       )}
       <MainImage
         {...(props as Omit<MainImageProps, "images" | "fallback">)}
+        className={imgClassName}
         {...getMainProps(
           isLoading,
           hasLoaded || isLoaded,
           images,
           loading,
           toggleIsLoaded,
           cacheKey,
-          ref
+          ref,
+          imgStyle
```

Comment:
```suggestion
          style: imgStyle
```

pretty sure your function expects `style`

Reply:
These are regular function arguments, not properties

Reply:
Ohh I see. When creating APIs that are used by others, I usually prefer named arguments, that simplifies extending that API and it avoids mixing up argument orders. I think this wraps it up pretty well: https://medium.com/@afontcu/cool-javascript-9-named-arguments-functions-that-get-and-return-objects-337b6f8cfa07

Or did you pick this API structure on purpose?

Reply:
I didn't write that function 😁